lib/teams/server/clan-message-actions.js

```diff
@@ -36,6 +36,15 @@ class ClanMessageActions
         if(TeamsConst.ACTIONS.CLAN_ACCEPTED === data.act && '1' === data.value){
             return await ClanJoin.execute(client, data, room, playerSchema, this.teamsPlugin);
         }
+        if(TeamsConst.ACTIONS.CLAN_ACCEPTED === data.act && '2' === data.value){
```

Comment:
data.value === 2 is never sent (unless you changed it on the client as well to send it), this event is never emited, just remove it if that's the case